pkg/model/dev.go

```diff
@@ -273,6 +277,12 @@ func Read(bytes []byte) (*Dev, error) {
 	return dev, nil
 }
 
+func (dev *Dev) loadName() {
+	if len(dev.Name) > 0 {
+		dev.Name = os.ExpandEnv(dev.Name)
```

Comment:
will this support env vars with a  default value ? `${foo-NAME}`  kind of thing.